common/api/core-common.api.md

```diff
@@ -666,6 +699,115 @@ export { BentleyError }
 
 export { BentleyStatus }
 
+// @alpha
+export class BetweenExpr extends BooleanExpr {
+    constructor(lhsExpr: ValueExpr, lowerBoundExpr: ValueExpr, upperBoundExpr: ValueExpr, not?: UnaryBooleanOp | undefined);
+    // (undocumented)
+    get children(): Expr[];
+    // (undocumented)
+    static deserialize(node: NativeECSqlParseNode): BetweenExpr;
+    // (undocumented)
+    readonly lhsExpr: ValueExpr;
+    // (undocumented)
+    readonly lowerBoundExpr: ValueExpr;
+    // (undocumented)
+    readonly not?: UnaryBooleanOp | undefined;
+    // (undocumented)
+    static parseOp(op: string): boolean[];
+    // (undocumented)
+    static readonly type = ExprType.Between;
+    // (undocumented)
+    readonly upperBoundExpr: ValueExpr;
+    // (undocumented)
+    writeTo(writer: ECSqlWriter): void;
+}
+
+// @alpha
+export class BinaryBooleanExpr extends BooleanExpr {
+    constructor(op: BinaryBooleanOp, lhsExpr: ComputedExpr, rhsExpr: ComputedExpr, not?: UnaryBooleanOp | undefined);
+    // (undocumented)
+    get children(): Expr[];
+    // (undocumented)
+    static deserialize(node: NativeECSqlParseNode): BinaryBooleanExpr;
+    // (undocumented)
+    readonly lhsExpr: ComputedExpr;
+    // (undocumented)
+    readonly not?: UnaryBooleanOp | undefined;
+    // (undocumented)
+    readonly op: BinaryBooleanOp;
+    // (undocumented)
+    readonly rhsExpr: ComputedExpr;
+    // (undocumented)
+    static readonly type = ExprType.BinaryBoolean;
+    // (undocumented)
+    writeTo(writer: ECSqlWriter): void;
+}
+
+// @alpha
+export enum BinaryBooleanOp {

```

Comment:
Did you consider just using a union for readability?

`if (op === BinaryBooleanOp.And || op === BinaryBooleanOp.GreaterThanOrEqualTo)`

vs

`if (op === "AND" || op == ">=")`